src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpContext.cs

```diff
@@ -374,159 +370,21 @@ public unsafe void SendResponseHeaders(bool appCompleted)
 
             HttpApiTypes.HTTP_RESPONSE_V2* pHttpResponse = NativeMethods.http_get_raw_response(_pHttpContext);
```

Comment:
No longer used.

Reply:
I will still leave the native method though it isn't used.

Reply:
Why?